languages/csharp/exercises/concept/properties/.meta/Example.cs

```diff
@@ -0,0 +1,59 @@
+using System;
+
+public enum Units
+{
+    Kilograms
+    ,Pounds
+}
+public class WeighingMachine
+{
+    private const float POUNDS_PER_KILOGRAM = 2.20462f;
+    private float inputWeight;
+
+    public Units Units { get; set; } = Units.Kilograms;
```

Comment:
I like this property, as it shows several things: auto-implemented properties, default values for properties and the fact that properties can be of any type.